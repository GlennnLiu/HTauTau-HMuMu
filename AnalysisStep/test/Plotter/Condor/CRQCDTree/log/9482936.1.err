mkdir: cannot create directory '/eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots': File exists
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_GGH_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_GGH_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_VBF_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_VBF_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_Boost_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_Boost_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_All_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_All_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_GGH_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_GGH_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_VBF_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_VBF_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_Boost_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_Boost_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_All_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mumu_All_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_GGH_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_GGH_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_VBF_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_VBF_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_Boost_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_Boost_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_All_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_All_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_GGH_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_GGH_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_VBF_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_VBF_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_Boost_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_Boost_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_All_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/etau_All_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_GGH_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_GGH_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_VBF_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_VBF_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_Boost_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_Boost_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_All_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_All_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_GGH_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_GGH_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_VBF_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_VBF_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_Boost_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_Boost_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_All_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/mutau_All_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_GGH_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_GGH_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_VBF_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_VBF_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_Boost_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_Boost_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_All_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_All_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_GGH_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_GGH_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_VBF_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_VBF_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_Boost_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_Boost_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_All_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautau_All_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_GGH_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_GGH_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_VBF_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_VBF_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_Boost_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_Boost_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_All_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_All_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_GGH_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_GGH_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_VBF_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_VBF_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_Boost_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_Boost_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_All_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/emu_All_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_GGH_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_GGH_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_VBF_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_VBF_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_Boost_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_Boost_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_All_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_All_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_GGH_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_GGH_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_VBF_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_VBF_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_Boost_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_Boost_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_All_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/tautaucomb_All_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_GGH_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_GGH_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_VBF_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_VBF_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_Boost_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_Boost_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_All_LLMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_All_LLMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_GGH_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_GGH_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_VBF_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_VBF_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_Boost_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_Boost_LLGoodMass.pdf has been created
Info in <TCanvas::Print>: png file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_All_LLGoodMass.png has been created
Info in <TCanvas::Print>: pdf file /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Plots/2l_All_LLGoodMass.pdf has been created
