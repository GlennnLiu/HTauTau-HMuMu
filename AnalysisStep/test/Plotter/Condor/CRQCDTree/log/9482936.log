000 (9482936.001.000) 07/27 15:12:38 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_8624_ecd7>
...
040 (9482936.001.000) 07/27 15:14:24 Started transferring input files
	Transferring to host: <128.142.61.36:9618?addrs=128.142.61.36-9618+[2001-1458-301-23--100-23]-9618&alias=b7s06p7054.cern.ch&noUDP&sock=slot1_13_4260_4e26_174177>
...
040 (9482936.001.000) 07/27 15:14:24 Finished transferring input files
...
001 (9482936.001.000) 07/27 15:14:25 Job executing on host: <128.142.61.36:9618?addrs=128.142.61.36-9618+[2001-1458-301-23--100-23]-9618&alias=b7s06p7054.cern.ch&noUDP&sock=startd_2986_b9e2>
...
006 (9482936.001.000) 07/27 15:14:34 Image size of job updated: 584144
	92  -  MemoryUsage of job (MB)
	94072  -  ResidentSetSize of job (KB)
...
040 (9482936.001.000) 07/27 15:14:43 Started transferring output files
...
040 (9482936.001.000) 07/27 15:14:43 Finished transferring output files
...
005 (9482936.001.000) 07/27 15:14:43 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:04, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:04, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	23792  -  Run Bytes Sent By Job
	342  -  Run Bytes Received By Job
	23792  -  Total Bytes Sent By Job
	342  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       37        1   1311536 
	   Memory (MB)          :       92     2000      2000 

	Job terminated of its own accord at 2023-07-27T13:14:43Z.
...
