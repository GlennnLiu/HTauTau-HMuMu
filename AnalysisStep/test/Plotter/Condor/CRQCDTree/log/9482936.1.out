[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.diBoson.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.triBoson.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.top.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.EWKZee.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.EWKZtt.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.EWKZmm.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.DYee.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.DYtt.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.DYmm.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Wjet.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.TTfake.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.Data.root
[INFO] Total MC summed
[INFO] Saved Histos to /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRQCDTree.TotalMC.root
